index.d.ts

```diff
@@ -16,6 +16,7 @@ interface WithUrqlInitialProps {
 
 interface PageProps {
   pageProps?: WithUrqlClient;
+  urqlClient: Client;
```

Comment:
This actually needs to go on the `NextContextWithAppTree`. `PageProps` is a special interface we use for the `_app.js` case, where the component defined in `_app.js` literally receives a prop called `pageProps` that has all of the wrapped Page's props inside it. So just move this line into the `NextContextWithAppTree` and we should be good!